quiz_hosting_web_app/quiz_management/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = "quiz_management.app.QuizAttempterConfig"
```

Comment:
Is this necessary for signals?

Reply:
Yes, this allows to register our signal automatically when app is loaded